app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    @user = User.find_by(email: params[:email])
+    if @user && !@user.activated? && @user.authenticated?(:activation_digest, params[:id])
+      @user.update_attribute(:activated, true)
+      @user.update_attribute(:activated_at, Time.zone.now)
```

Comment:
tạo 1 method ở model có tên là `activate`, rồi gọi ở controller
_Note_: sử dụng `update_columns`, không dùng 2 lệnh  `update_attribute`(a đã giải thích)
```suggestion
      @user.activate
```